doc/install/index.rst

```diff
@@ -9,3 +9,4 @@ Installation
    install-lammps
    install-ipi
    build-conda
+   install-gromacs
```

Comment:
It should be located before `build-conda`.

Reply:
Exactly...

Reply:
not resolved yet?